docs/source/development/coding_conventions.rst

```diff
@@ -292,6 +292,89 @@ If this is a link it should refer to where the link goes.
 	<element id="mds.nav.schema-editor"/>
 	<element id="task.sidebar.tasks-toggle-active"/>
 
+
+Common Conventions for API creation
+###################################
+
+HTTP methods and endpoints
+--------------------------
+
+We use following common pattern for endpoints:
+
+``http://<motech-server-address>/module/{module-name}/<<path>>``
+
+|v| **Do** use following general pattern for <<path>>:
+
+``/{resource}``
+
+``/{resource}/{path-variable}``
+
+Path can take params.
+
+|v| **Do** use HTTP methods verbs:
+
+- GET for read resource by id or collection.
+- GET for read - lookup resource by id or collection.
```

Comment:
lookup is a specific thing in MDS, this coding conventions should have rather general character